// docz-utils/lib/index.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

require('@babel/parser');
require('@babel/traverse');
var ast = require('./ast.js');
require('prettier');
require('signale');
var format = require('./format.js');
require('path');
require('fs-extra');
require('art-template');
var fs$1 = require('./fs.js');
var get = _interopDefault(require('lodash/get'));
var mergeWith = _interopDefault(require('lodash/fp/mergeWith'));
require('@babel/generator');
var imports = require('./imports.js');
var exports$1 = require('./exports.js');
require('jsx-ast-utils');
require('strip-indent');
require('js-string-escape');
var jsx = require('./jsx.js');
require('to-vfile');
require('unified');
require('remark-parse');
require('remark-frontmatter');
require('remark-slug');
require('remark-parse-yaml');
require('unist-util-find');
require('unist-util-visit');
require('humanize-string');
require('lodash/flatten');
var mdast = require('./mdast.js');



exports.codeFromNode = ast.codeFromNode;
exports.valueFromTraverse = ast.valueFromTraverse;
exports.format = format.format;
exports.compiled = fs$1.compiled;
exports.touch = fs$1.touch;
exports.get = get;
exports.mergeWith = mergeWith;
exports.getFullImports = imports.getFullImports;
exports.getImportsVariables = imports.getImportsVariables;
exports.getExportsVariables = exports$1.getExportsVariables;
exports.componentName = jsx.componentName;
exports.propFromElement = jsx.propFromElement;
exports.removeTags = jsx.removeTags;
exports.sanitizeCode = jsx.sanitizeCode;
exports.getParsedData = mdast.getParsedData;
exports.headingsFromAst = mdast.headingsFromAst;
exports.parseMdx = mdast.parseMdx;
